packages/kit/src/index.tsx

```diff
@@ -1,61 +1,17 @@
-import { DropdownButton } from './DropdownButton'
-import { Form, SubmitButton, TextField } from './Form'
-import { Icon } from './Icon'
-import { LongPressSheetButton } from './LongPressSheetButton'
-import { QRCode } from './QRCode'
-import { SelectField } from './SelectField'
-import { Slider, SliderField } from './Slider'
-import { SortableList } from './SortableList'
-import { SwitchField } from './SwitchField'
+import QRCode from 'react-native-qrcode-svg'
+
+import { RNDraggableFlatList } from './client/DraggableFlatList'
+import { LucideIcon } from './client/Icon'
 
-// tbd: enforce same components are exported
 export const RiseComponents = {
-  RiseQRCode: {
-    component: QRCode,
-    validator: QRCode.validate,
-  },
-  RiseIcon: {
-    component: Icon,
-    validator: Icon.validate,
-  },
-  RiseSelectField: {
-    component: SelectField,
-    validator: SelectField.validate,
-  },
-  RiseDropdownButton: {
-    component: DropdownButton,
-    validator: DropdownButton.validate,
-  },
-  RiseSlider: {
-    component: Slider,
-    validator: Slider.validate,
+  RNDraggableFlatList: {
+    component: RNDraggableFlatList,
   },
-  RiseSliderField: {
-    component: SliderField,
-    validator: SliderField.validate,
-  },
-  RiseSortableList: {
-    component: SortableList,
-    validator: SortableList.validate,
-  },
-  RiseSwitchField: {
-    component: SwitchField,
-    validator: SwitchField.validate,
-  },
-  RiseForm: {
-    component: Form,
-    validator: Form.validate,
-  },
-  RiseTextField: {
-    component: TextField,
-    validator: TextField.validate,
-  },
-  RiseSubmitButton: {
-    component: SubmitButton,
-    validator: SubmitButton.validate,
+  RNQRCode: {
+    component: QRCode,
```

Comment:
here's a funny thought, if we support SVGs we can probably render the QRCode on the server (although that might make the payload too large)

Reply:
Yeah, some random stats from the internet:
```
JPEG, 90%: 39.4 kB
JPEG, 50%: 25,7 kB
PNG: 870 Bytes
GIF: 6,8 kB
SVG: 29 kB
SVG, gzipped: 2,4 kB
```
PNG would be cool, actually!